platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblems.kt

```diff
@@ -202,7 +202,6 @@ class ConfigurationCacheProblems(
                 message,
                 configCacheValidation
             )
-//            stackLocation()
```

Comment:
🤔 Why don't the configuration cache problems add a stacktrace location?

Reply:
There is one coming from the configuration cache problems, they are already created by using stacktraces, we'd do the same work twice essentially.